strawberryfields/ops.py

```diff
@@ -1525,33 +1525,40 @@ def __str__(self):
         return 'Del'
 
 
-class New_modes(MetaOperation):
-    """Used for adding new modes to the system.
-    Also accessible via the shortcut variable ``New``.
+def New(n=1):
+    """Adds new subsystems to the quantum register.
+
+    The new modes are prepared in the vacuum state.
 
-    The new modes are prepapred in the vacuum state.
+    Must only be called in a :class:`Program` context.
 
-    This class cannot be used with the :meth:`__or__` syntax since it would be misleading,
-    instead we use :meth:`__call__` on a single instance to dispatch the command to the :class:`.Program`.
+    Args:
+        n (int): number of subsystems to add
+    Returns:
+        tuple[RegRef]: tuple of the newly added subsystem references
     """
-    ns = 0
+    # create RegRefs for the new modes
+    refs = Program._current_context._add_subsystems(n)
+    # append the actual Operation to the Program
+    Program._current_context.append(New_modes(n), refs)
+    return refs
+
 
-    def __call__(self, n=1):
-        """Adds one or more new modes to the system in a deferred way.
+class New_modes(MetaOperation):
```

Comment:
Since these are not user facing, does it make sense to use underscores so that they are 'private' i.e., `_NewModes`

Reply:
(I could imagine a use case where the user is in IPython or Jupyter notebook, and types `ops.N`, and `New_modes` comes up as an option)

Reply:
Maybe? The only reason not to use underscores that I can think of is that they ever so slightly make the names longer and clutter up the program listing.

Reply:
I'll add the underscores, it's probably safer that way.